lib/plugins/aws/package/compile/events/stream/index.js

```diff
@@ -191,8 +191,14 @@ class AwsCompileStreamEvents {
               );
             }
 
+            const streamResource = JSON.parse(streamTemplate);
+
+            if (typeof event.stream.batchWindow !== 'undefined') {
```

Comment:
More natural would be: `if (event.stream.batchWindow)`

Reply:
Ahh yep. Will update.